pot_ranges: {pot0: 173.0, pot1: 186.0, pot2: 163.0, pot3: 176.0, pot4: 164.0, pot5: 184.0}

rosdistro: 'melodic

  '
roslaunch:
  uris: {host_robothand_hp_elitebook_840_g1__35523: 'http://robothand-HP-EliteBook-840-G1:35523/',
    host_robothand_hp_elitebook_840_g1__39951: 'http://robothand-HP-EliteBook-840-G1:39951/'}
rosversion: '1.14.13

  '
run_id: 5063ba5c-f0dd-11ec-9410-2babb065445d
serial_node: {baud: 9600, port: /dev/ttyACM0}
